magento/magento2
# Switching between Media Gallery bookmarks does not change the selected folder

### Description (*)

This PR will implement the fix to change the selected folder when switching views in Media Gallery.


### Related Pull Requests

N/A
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1523

### Manual testing scenarios (*)


Apart from the mentioned steps please check directory navigation/operations in general

1. Go **Content** - Pages click **Add New Page**
2. Expand **Content** click **Show / Hide Editor**, **Insert Image**...
3. Create a main folder named _folder_ for example and select it
4. Click on **Default View** dropdown
5. Click **Save View As**..., and click **Save all changes** ( ➡️ )
 _now you have a New View with the selected folder_
7. Close the Media Gallery and go to **Catalog - Categories**
8. Expand Content and click Select from Gallery
_folder is selected_
10. Try to switch between **New View** and **Default View**
11. The folder selected for that view must be highlighted/selected,
 i.e.: If **New View** is selected, _folder_ should be selected,
**Catalog/category** folder is selected if the **Default view** is selected

### Questions or comments


### Contribution checklist (*)
